### PR TITLE
Add new snippet

### PR DESCRIPTION
**Pull request by antoBit**
Added a new snippet for using switch statements with numeric ranges, I always found this useful :)

### CODE REVIEW

File: snippets/24-use-switch-case-with-numeric-ranges.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Minutes of Javascript #24 - Use switch/case statement with numeric ranges
+
+//You can use a switch statement with a numeric range in the case with:
```
Comment (olivierloverde) [inline snippets/24-use-switch-case-with-numeric-ranges.js]:
```suggestion
// switch (true) may seem absurd but you can match against values as well as expressions. 
// Expressions will be evaluated before matching, so any expression that evals to true will be matched.
// Ref: https://262.ecma-international.org/10.0/#sec-switch-statement

// For example, you can use a switch statement with a numeric range in the case with:
```

File: snippets/24-use-switch-case-with-numeric-ranges.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Minutes of Javascript #24 - Use switch/case statement with numeric ranges
```
Comment (olivierloverde) [inline snippets/24-use-switch-case-with-numeric-ranges.js]:
```suggestion
// Minutes of Javascript #24 - Usage of switch (true), an example with numeric ranges
```

File: README.md [hunk 3]
```diff
@@ -65,6 +65,7 @@ Javascript pieces of code to make you discover new exciting features, useful tip
 21. [Using "_" in numbers for better readability](snippets/21-tips-underscore-number.js)
 22. [Array: .map() vs .filter() vs .reduce()](snippets/22-map-vs-filter-vs-reduce.js)
 23. [Generator function* and yield usage](snippets/23-generator-function-yield-usage.js)
+24. [Use switch/case statements with numeric ranges](snippets/24-use-switch-case-with-numeric-ranges.js)
```
Comment (olivierloverde) [inline README.md]:
```suggestion
24. [Usage of switch (true), an example with numeric ranges](snippets/24-use-switch-case-with-numeric-ranges.js)
```

File: snippets/24-use-switch-case-with-numeric-ranges.js (hunk 1 above)
Comment (antoBit) [reply snippets/24-use-switch-case-with-numeric-ranges.js]: Ah, this description makes so much more sense than mine, thanks!! 👍🏻

**Comment by olivierloverde** [comment]
> Added a new snippet for using switch statements with numeric ranges, I always found this useful :)

Thanks a lot for contributing @antoBit , and welcome here! :)

I've suggested a few changes, could you please have a look?

**Comment by antoBit** [comment]
I was just looking at them, great additions, thank you! I'll merge them :)

**Comment by antoBit** [comment]
@olivierloverde I updated the filename as well, let me know if something isn't ok :)

**Comment by olivierloverde** [comment]
That's great work, thanks a lot, merging :)

**Pull request merged**